integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/BearerTokenAuthorizationTest.java

```diff
@@ -85,7 +84,7 @@ public void testAccessResourceAzure() throws Exception {
 
     private String readFile(String filePath) throws Exception {
         try (InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(filePath)) {
-            byte[] content = FileUtil.readFileContents(is);
```

Comment:
Because of this static import of the same code `is.readAllBytes();` the deployment dependency was needed here.